src/admin/fusionauth/fusionauth.service.ts

```diff
@@ -67,13 +71,12 @@ export class FusionauthService {
   getUsers(
     applicationId: string,
     startRow: number,
-    numberOfResults: number
-  ): Promise<{ total: number, users: Array<User> }> {
+    numberOfResults: number,
+  ): Promise<{ total: number; users: Array<User> }> {
     const searchRequest = {
       search: {
         numberOfResults: numberOfResults,
-        query:
-          `{"bool":{"must":[{"nested":{"path":"registrations","query":{"bool":{"must":[{"match":{"registrations.applicationId":"${applicationId}"}}]}}}}]}}`,
+        query: `{"bool":{"must":[{"nested":{"path":"registrations","query":{"bool":{"must":[{"match":{"registrations.applicationId":"${applicationId}"}}]}}}}]}}`,
```

Comment:
Create a query generator service and move this to there. Right now it would be difficult to make sense of what is happening here. Also keep the object as JSON and use `JSON.stringify` to create the query.